src/index.js

```diff
@@ -120,6 +168,42 @@ export default class ImgixClient {
     }
   }
 
+  /**
+   * _buildSrcSet static method allows full URLs to be used when generating
+   * imgix formatted `srcset` string values.
+   *
+   * - If the source URL has included parameters, they are merged with
+   * the `params` passed in as an argument.
+   * - URL must match `{host}/{pathname}?{query}` otherwise an error is thrown.
+   *
+   * @param {String} url - full source URL path string, required
+   * @param {Object} params - imgix params object, optional
+   * @param {Object} options - imgix client options, optional
```

Comment:
That ms. 

I wouldn't be against adding parameters to this method to separate client options from sourceset modifiers.

So that instead of

```js 
_buildSrcSet(url, params = {}, options = {})
```

The arguments could be something more like

```js
_buildSrcSet(url, params = {}, srcsetModifiers = {}, clientOptions = {})
```

Obv we'd update the js-doc to match, something like

```js
/**
   * @param {String} url - full source URL path string, required
   * @param {Object} params - imgix params object, optional
   * @param {Object} srcsetModifiers - srcset modifiers, optional
   * @param {Object} clientOptions - imgix client options, optional
*/
```

Would you be in favor of introducing those params @sherwinski ?

Reply:
Yeah that would be most clear imo 👍